core/blockchain_test.go

```diff
@@ -17,7 +17,6 @@
 package core
```

Comment:
I am not quite comfortable with some of the tests that are eliminated here. Verifying that the code works correctly in the case of forks and reorgs is important even though we don't expect it to happen unless something has gone wrong.